app/utils.py

```diff
@@ -104,14 +135,16 @@ def gen_data(ckan, pids):
             status = 'Invalid frequency. Status could not be determined.'
 
         data = {
-            'dataset_id': pid,
+            'dataset_id': package['id'],
```

Comment:
`pid` was returning package `name` which is interchangeable with package id in CKAN. By package id I mean a package hash (i.e.  `119556f4-9c5f-40f2-881c-d48ae4aa60be`)